filebeat/input/docker/input.go

```diff
@@ -37,6 +40,10 @@ func init() {
 	}
 }
 
+var (
+	logger = logp.NewLogger("docker")
```

Comment:
Move this into `NewInput`. See the documentation of `NewLogger` for the reason.

Reply:
makes sense why i didnt see that log message lol :)